app/containers/Forms/ProductField.tsx

```diff
@@ -86,14 +86,14 @@ export const ProductFieldComponent: React.FunctionComponent<Props> = (
       <strong>Warning:</strong> This Product or Service is not associated with
       the NAICS code reported in this application. Please review the guidance
       documents for a list of the valid products for this sector or verify the
-      NAICS code reported in the Administration data is correct.
+      NAICS code reported in the administrative data is correct.
```

Comment:
```suggestion
      NAICS code reported in the Administrative Data is correct.
```

Reply:
Probably best to keep the capitalized convention to ensure they know we're referring to a form instead of some other generic concept. "Data" should be capitalized as well so it matches the form tab.

Ditto for the message below